Resources/ALC230/Info.plist

```diff
@@ -39,113 +55,149 @@
 			<key>Count</key>
 			<integer>1</integer>
 			<key>Find</key>
-			<data>QcYGAEmLvCQ=</data>
```

Comment:
Please fix xml formatting here and below. AppleALC rebuild should do.

Reply:
Fixed ! Thanks 👍